doc_source/studio-tasks-git.md

```diff
@@ -16,12 +16,12 @@ Amazon SageMaker Studio can connect only to a local repository\. In this example
 
 1. Choose **CLONE**\.
 
-1. If the repo requires credentials, you are prompted to enter your username and password\.
+1. If the repo requires credentials, you are prompted to enter your username and password/personal access token. `From Aug 13, 2021 password does not work and you need to use a personal access token.`
```

Comment:
Suggested edit:
"If the repo requires credentials, you are prompted to enter your username and personal access token."

Reply:
Done.